profiles.md

```diff
@@ -0,0 +1,236 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This repository allows to manage the profile item linked to a particular eperson. The specific type of item 
+with which the profile is modeled depends on the dspace configuration (by default it is the Person type). If the particular dspace instance doesn't handle the "profile" 
+type then these endpoints are disabled.
+
+## Get all Profile
+**GET /api/eperson/profiles/**
+
+This operation is not currently supported.
+
+## Single Profile
+**GET /api/eperson/profiles/<:eperson-uuid>**
+
+Provide information about a specific profile. Only the Administrators or the owner eperson can access this endpoint.
+
+The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "visible": true,
+  "type": "profile"
+}
+```
+
+Attributes:
+* id: the UUID of the EPerson who owns the profile. This also corresponds to the ID of the profile.
+* visible: this property provides a quick way to add/remove the Anonymous READ policy. When set to "true", then the Anonymous READ Policy is added (and the profile becomes accessible to anonymous users). When set to "false", then the Anonymous READ policy is removed (and the profile is no longer accessible anonymously).
```

Comment:
It seems my previous feedback is no longer associated with this line, and not very visible anymore

I would recommend to further clarify this since this is a very atypical property. This can be done by stating the permissions are unknown if `visible=false`. It may be accessible to many users, or admins only, or also the EPerson itself (if they're granted READ permissions), …. The only thing that's known is whether there's an Anonymous READ policy without start/end date

Reply:
@benbosman : To resolve this, maybe we add a bit of clarification on the end of the existing description? 

So, for example, right after:
`When set to "false", then the Anonymous READ policy is removed (and the profile is no longer accessible anonymously).`
Maybe we clarify it with:
`Keep in mind, this setting only impacts the Anonymous READ policy. So, if other custom READ policies exist on the profile, those policies will take effect when "visible=false".`